src/admin/systemTypes/systemTypes.component.tsx

```diff
@@ -43,6 +69,16 @@ function SystemTypes() {
         columnFilterModeOptions: COLUMN_FILTER_MODE_OPTIONS.string,
         Cell: ({ row }) => row.original.value,
       },
+      {
+        header: 'Is Spare',
```

Comment:
```suggestion
        header: 'Contains spares',
```
I feel emphasising the systems with the type contains them is clearer?

Reply:
I agree with changing the column name, however `Contains spares` is misleading because it suggests the system type might include other kinds of items as well.

I asked Copilot for some suggestions:

```
Spare-Only Type  
Allows Only Spare Items  
Restricted to Spares  
Spare Item Type  
Spare-Exclusive System Type  
Spare Items Only  
Is Spare-Only?  
Spare Type (Yes/No)  
System Type for Spares  
Spare Use Only
```

I like the last two. what is you preference from the list?

Reply:
Yeah out of those I lean towards the last two. Perhaps a slight rephrasing e.g. "Used for spares"?